auth.go

```diff
@@ -733,7 +733,14 @@ func (x *Central) StartMergeTicker() error {
 	}
 	x.syncMergeTickerStopRequest = make(chan bool)
 	x.syncMergeTickerStopResponse = make(chan bool)
+
 	go func() {
+		defer func() {
+			if r := recover(); r != nil {
+				x.Log.Errorf("MergeTick panic: %v", r)
```

Comment:
So in mm/rad we have this whole `GetStack` function that we use when logging the error. Does the `r` have enough info that we don't need to dump the stack?

Here is the `GetStack` function for reference
```go
func GetStack() string {
	buf := make([]byte, 1024)
	for {
		n := runtime.Stack(buf, false)
		if n < len(buf) {
			return string(buf[:n])
		}
		buf = make([]byte, 2*len(buf))
	}
}
```

Reply:
Good call...will implement and test.

Reply:
Was not enough info initially (only high level error message, e.g. `division by zero`). Will implement GetStack as is...